README.md

```diff
@@ -1 +1,85 @@
-Test
\ No newline at end of file
+# Trustly Parliament Challenge
+
+## Problem
+
+This is a challenge by Trustly known as the **"Parliament Challenge"**.
+
+The Swedish parliament has an open API platform for developers to use its resources to create applications. Among its resources are the speeches in the parliament and the data of its members.
+
+The challenge by Trustly is to create an appliction that serves requests for parliament speeches. The expected response should be created by merging the data from the speech's api and member's api and should be in a JSON format.
+
+The expected response should at least contain:
+
+- A unique ID, representing the speech itself (anforande_id - speech's api)
+- Date of speech (dok_datum - speech's api)
+- The name of the speaker, and only the name (tilltalsnamn - member's api)
+- Political affiliation (parti - speech's api)
+- The official e-mail address (uppgift - member's api)
+- Constituency (valkrets - member's api)
+- A decent sized image that could be used in a web site listing (bild_url_192 - member's api)
+- The debate subject (avsnittsrubrik - speech's api)
+- A link to the speech (HATEOAS style) (protokoll_url_www - speech's api)
+
+The link to the challenge can be found on https://github.com/trustly/parliament-challenge
+
+## Approach
+
+### Speech's API
+
+The speech's api represents the details of the speeches held at the Swedish parliament. This api has a filter option to get the number of speeches that the client requires. In this challenge, the objective is to get the ten latest speeches. The relevant keys and values that this api has are: **anforande_id, dok_datum, parti, avsnittsrubrik and protokoll_url_www**.
+
+Link to the speech's api: http://data.riksdagen.se/anforandelista/?anftyp=Nej&sz=10&utformat=json
+
+### Member's API
+
+The member's api represents the details of the members of the Swedish parliament. In order get the relevant member data and link the speech to a member, a reference key **intressent_id** (which is common in both apis) is used as a filter option. The relevant keys and values that this api has are: **tilltalsnamn, valkrets, bild_url_192**.
+
+Link to the member's api: http://data.riksdagen.se/personlista/?iid=&utformat=json
+
+### Merge
+
+In order to merge a speech to a member, the relevant keys and values are filtered from the the speech api and the reference key **intressent_id** from a speech is passed as an arguement and parameter to get the relevant (filtered) member data which is then updated with that relevant speech data and stored in a list of dict.
```

Comment:
This definition is too detailed. Make it simpler. Describe it in plain english without any technical implementation details. Explain it like as if you're explaining it to a non-technical person.

Reply:
You don't have to say which functions you're calling and what arguments you pass to it. The programmer or anyone reading your code will understand it from reading the code. The description should only be very high level .. in plain english what the function is supposed to do, what arguments it takes and what it returns.

Reply:
updated explanation: 

In order to merge a speech to a member, the function will filter the respective apis for the relevant keys and values and link a speech to a member through the reference key **intressent_id**. The function will then update the speech data with member data by combining both data sets together and present a merged data.